openforcefield/utils/toolkits.py

```diff
@@ -247,6 +255,16 @@ def is_available():
         """
         return NotImplementedError
 
+    @property
+    def toolkit_version(self):
+        """
+        Return the version of the wrapped toolkit
```

Comment:
Can we say anything about the type that's returned here? If we can hold ourselves to it, maybe specify that it'll always be `str`, and add that to the test so future developers know if they've broken that contract.

Reply:
Generally it's strings all the way down, but I didn't implement a version for `BuiltInToolkitWrapper`. Blocking, or should it default to something like `'0.0.0'` since we're probably not interested in versioning it?

```
In [4]: [(type(key), type(val)) for key, val in GLOBAL_TOOLKIT_REGISTRY.registered_toolkit_versions.items()]
Out[4]: [(str, str), (str, str), (str, str), (str, NoneType)]